LiskHQ/lisk-desktop
# Update logo to the latest version - Closes #328

### What was the problem?
The logo was not the latest version.
### How did I fix it?
Updated logo to the latest version.
### How to test it?
Open app, open menu, try different breakpoints.

`npm run dist` and open the generated `*.dmg` to check the icon.

### Review checklist
- The PR solves #328
- All new code is covered with unit tests
- All new features are covered with e2e tests
- All new code follows best practices
